packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -850,15 +903,25 @@ class Scaffold extends StatefulWidget {
   ///  * [showModalBottomSheet], which displays a modal bottom sheet.
   final Widget bottomSheet;
 
-  /// Whether the [body] (and other floating widgets) should size themselves to
-  /// avoid the window's bottom padding.
+  /// This flag is deprecated, please use resizeToAvoidBottomInset
```

Comment:
`[resizeToAvoidBottomInset]` with square brackets to directly link to it?

Reply:
Yes, done.